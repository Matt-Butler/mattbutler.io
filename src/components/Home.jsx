import React from 'react';
import logo from './logo.svg';
import './Home.css';

const Home = () => (
  <div className="Home">
    <div className="Home-header">
      <img src={logo} className="Home-logo" alt="logo" />
      <h2>mattbutler.io</h2>
    </div>
  </div>
);

export default Home;
