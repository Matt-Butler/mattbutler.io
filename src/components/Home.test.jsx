import React from 'react';
import toJson from 'enzyme-to-json';
import { shallow } from 'enzyme';
import Home from './Home';

it('matches snapshot', () => {
  const wrapper = shallow(<Home />);

  expect(toJson(wrapper)).toMatchSnapshot();
});
