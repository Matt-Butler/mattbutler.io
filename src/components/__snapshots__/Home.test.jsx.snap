// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`matches snapshot 1`] = `
<div
  className="Home"
>
  <div
    className="Home-header"
  >
    <img
      alt="logo"
      className="Home-logo"
      src="logo.svg"
    />
    <h2>
      mattbutler.io
    </h2>
  </div>
</div>
`;
